boards/arm/nrf52833dk_nrf52820/nrf52833dk_nrf52820.yaml

```diff
@@ -0,0 +1,20 @@
+identifier: nrf52833dk_nrf52820
+name: nRF52833-DK-NRF52820
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 32
+flash: 256
+supported:
+  - arduino_gpio
```

Comment:
Since `arduino_header` is not defined in dts for this board, this entry should not be present here.